lib/renderer/web-view/web-view-constants.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   ATTRIBUTE_SRC: 'src',
   ATTRIBUTE_HTTPREFERRER: 'httpreferrer',
   ATTRIBUTE_NODEINTEGRATION: 'nodeintegration',
+  ATTRIBUTE_DISABLEREMOTEMODULE: 'disableremotemodule',
```

Comment:
The `webPreferences` property is `enableRemote` but the webview attribute is `disableremotemodule`?

Can we be consistent in both the property, attribute and use in our code.  I.e. If the property is called `enableRemote` don't have a method called `IsRemoteDisabled()`.  Either use `disable` or `enable` and write the logic around that consistently.

Reply:
I wanted to call it `disableRemoteModule`, but @alexeykuzmin said that having a negative there is weird and that he would prefer calling it `enableRemoteModule`. But with `<webview>`, it got complicated as I would have to have `enableremotemodule="false"`, which makes it inconsistent with other attributes.

Reply:
@MarshallOfSound `enable...` naming is used everywhere except two cases:
1. A `<webview>` element's attribute. It is ok for HTML attributes to not have a value (e.g. `<input disabled>`), and we already follow that pattern having `disableblinkfeatures` and `disablewebsecurity` attributes of `<webview>`.
2. A command line switch. Boolean switches often come in pairs: `--enable-something` and `--disable-something`.

Reply:
@alexeykuzmin If that's the case can we immediately map `disableremotemodule` to `!enableRemote`.  The smallest amount of opposite logic possible would be great, would prefer if it was `enableremotemodule="false"` but really just want this to be as non-confusing as possible

Reply:
done